libndt-client.cpp

```diff
@@ -229,5 +229,9 @@ int main(int, char **argv) {
     client.reset(new libndt::Client{settings});
   }
   bool rv = client->run();
+  if (rv) {
+    std::cout << std::endl;
+    client->summary();
```

Comment:
It looks like the summary is unconditionally printed. This means that if there are systems that expect the output to be JSONL (like it used to be, I think?) then they will fail on this new output.  I will click :LGTM: here, because I am not writing any of the parsers for the client output, so I don't know whether that is the case or not.

Reply:
I've removed the summary in BatchClient. However, your concern about mixed JSON and non-JSON output still makes a lot of sense as there are still several non-JSON messages.

The way I had implemented `-batch` (trying to minimize changes until I got more confident with the codebase) makes it so that all the JSON output is printed to STDOUT. I did not change the logging macros, however, which all use `std::clog` and thus print to STDERR. Tools calling libndt-client with `-batch`, like Murakami, should not try to interpret STDERR as JSONL.

E.g. these commands only print valid JSONL:
```
./libndt-client -download -upload -batch 2> /dev/null
./libndt-client -download -upload -batch -ndt7 2> /dev/null
```